app/Http/Enums/MajorIdentifications.php

```diff
@@ -29,6 +29,8 @@ enum MajorIdentifications
    case EXPLOSIVE_IMPACT;
    case GEOCENTRISM;
    case GRAVITYWELL;
+   case DESC_SNOWYSTEPS;
```

Comment:
Shouldn't this  be `SNOWY_STEPS`

Reply:
![image](https://user-images.githubusercontent.com/57310593/208243201-f29d7405-854d-4f86-8e4b-f26e9c9fba30.png)

Doesn't look like it

Reply:
Alright, thanks for clarifying!